### PR TITLE
feat: add pangolin call table

### LINKED ISSUE #166: Let Pangolin call on normal- AND on pseudo-assembly

### LINKED ISSUE #211: Check if the pangolin call on the pseudoassembly is working correctly

### PR DESCRIPTION
**Pull request by thomasbtf**
# Description

Adds a table, which contains pangolin of various stages throughout the workflow (scaffolded-, polished-, masked polished-, pseudo-, consensus-, masked consensus-sequences)

## Related Issue

closes #211
closes #166

## Checklist

- [X] I've formatted the PR title in accordance with the [structure of the conventional commit messages](https://www.conventionalcommits.org/en/v1.0.0/).
- [X] I've updated the code style using `snakefmt workflow/` and `black workflow/` if needed.
- [x] I've updated or supplemented the documentation as needed.
- [X] I've read the [`CODE_OF_CONDUCT.md`](https://github.com/IKIM-Essen/uncovar/blob/master/CODE_OF_CONDUCT.md) document.
- [X] I've read the [`CONTRIBUTING.md`](https://github.com/IKIM-Essen/uncovar/blob/master/CONTRIBUTING.md) guide.

### CODE REVIEW

**Review by alethomas — APPROVED** [review]

File: docs/consumer-guide/index.md [hunk 1]
```diff
@@ -1,9 +1,11 @@
 # Consumer Guide
 
-This section of the documentation is focused on interpreting the results produced by UnCoVar.
+This section of the documentation is focused on final data interpreted,
```
Comment (alethomas) [inline docs/consumer-guide/index.md:3]: These lines sound difficult

File: docs/consumer-guide/report-structure.md [hunk 2]
```diff
@@ -1 +1,60 @@
...
+[*de novo* assembly](https://en.wikipedia.org/wiki/De_novo_sequence_assemblers),
+ordered against the virus reference genome (default: Wuhan-Hu-1 Reference
+[`NC_045512.2`](https://www.ncbi.nlm.nih.gov/nuccore/1798174254)) with 
+[RaGOO](https://github.com/malonge/RaGOO).
+- **Polished** Sequences: Scaffolded sequence polished by applying variants with
+an allele frequency of 100% (called by [Varlociraptor](https://varlociraptor.github.io)
+at FDR `5%` by default).
+- **Masked Polished** Sequences: Polished Sequences masked due to two criteria.
+See [below](#quality-criteria) for a more in-depth explanation of these two criteria.
+- **Consensus** Sequences: Using corrected reads and the the virus reference
```
Comment (alethomas) [inline docs/consumer-guide/report-structure.md:31]: Double the

File: docs/consumer-guide/report-structure.md [hunk 3]
```diff
@@ -1 +1,60 @@
...
+genome (default: Wuhan-Hu-1 Reference 
+[`NC_045512.2`](https://www.ncbi.nlm.nih.gov/nuccore/1798174254)) a
+[consensus sequence](https://en.wikipedia.org/wiki/Consensus_sequence)
+is generated using [medaka](https://github.com/nanoporetech/medaka).
+Consensus Sequences are only generated for Oxford Nanopore data.
+- **Masked Consensus** Sequences: Consensus Sequence masked due to low position
+coverage. See [below](#quality-criteria) for a more in-depth explanation of the criteria.
+Masked consensus Sequences are only generated for Oxford Nanopore data.
+- **Pseudo** Sequences: Sequences that generated based on the virus reference
```
Comment (alethomas) [inline docs/consumer-guide/report-structure.md:40]: 'are' missing

**Review by alethomas — CHANGES_REQUESTED** [review]

File: docs/consumer-guide/report-structure.md (hunk 3 above)
Comment (thomasbtf) [reply docs/consumer-guide/report-structure.md:40]: :+1

File: docs/consumer-guide/report-structure.md (hunk 2 above)
Comment (thomasbtf) [reply docs/consumer-guide/report-structure.md:31]: :+1

File: docs/consumer-guide/index.md (hunk 1 above)
Comment (thomasbtf) [reply docs/consumer-guide/index.md]: Thanks! See cd7cd99

File: docs/consumer-guide/report-structure.md (hunk 2 above)
Comment (thomasbtf) [reply docs/consumer-guide/report-structure.md]: Thanks! See cd7cd99

File: docs/consumer-guide/report-structure.md (hunk 3 above)
Comment (thomasbtf) [reply docs/consumer-guide/report-structure.md]: Thanks! See cd7cd99

File: docs/consumer-guide/index.md [hunk 4]
```diff
@@ -1,10 +1,11 @@
 # Consumer Guide
 
-This section of the documentation is focused on interpreting the
-results produced by UnCoVar.
+This section of the documentation is focused on data interpretation. Its target
+group consists of virologists, epidemiologists or other scientists. In general,
+guidance for interpreting the results produced by UnCoVar is provided.
```
Comment (alethomas) [inline docs/consumer-guide/index.md:5]:
```suggestion
This section of the documentation is focused on data interpretation, targeting
virologists, epidemiologists and other involved scientists. General guidance for
interpreting the results produced by UnCoVar is provided
```

**Review by alethomas — CHANGES_REQUESTED** [review]

**Pull request merged**